### DISCUSSION TITLE
Update speed?

### DISCUSSION THREAD

**Discussion by muhkuh666** [macxq/foxess-ha#149] [Q&A]
I wonder how the update speed to Foxess is set ?

**Comment by vicdavery** [comment]
Why do you ask? 
The default is to update every 5 mins. I currently have some issue where it is only updating once per hour. This is a recent problem. 
I can see that HA and the integration are requesting the info every 5 minutes as expected, but the values only change once per hour.

**Reply by muhkuh666** [reply]
I roughly every 17 minutes... so slow...

**Comment by stevetrease** [comment]
In my Home Assistant logs I see the integration getting new data every five minutes, however the values are not changing very often.

For example looking at SoC history in HA appears to be updating about once an hour.

Using the FoxCloud iOS app the battery charge percent is changing more often than that. 

Is this correct behaviour?

Thanks,

Steve

**Reply by vicdavery** [reply]
This is the behaviour I am currently seeing. However, that is not how it used to be. Every 5 mins new values would be received. 
I don't know what has changed, but I _think_ it is on the FoxESS side rather than the HA integration.
